packages/@ama-sdk/swagger-builder/package.json

```diff
@@ -10,14 +10,14 @@
     "nx": "nx",
     "ng": "yarn nx",
     "build": "yarn nx build ama-sdk-swagger-builder",
-    "postbuild": "patch-package-json-main",
+    "postbuild": "patch-package-json-main && cpy 'src/schemas/*.json' 'dist/schemas'",
```

Comment:
is `src/schemas/*.json` already included in Nx task outputs?

Reply:
I want to remove the src folder from the dist. Hence, the one from nx will be removed.